core/lpms.go

```diff
@@ -129,7 +130,15 @@ func (l *LPMS) HandleHLSPlay(
 //SegmentRTMPToHLS takes a rtmp stream and re-packages it into a HLS stream with the specified segmenter options
 func (l *LPMS) SegmentRTMPToHLS(ctx context.Context, rs stream.RTMPVideoStream, hs stream.HLSVideoStream, segOptions segmenter.SegmenterOptions) error {
 	//Invoke Segmenter
-	localRtmpUrl := "rtmp://localhost" + l.rtmpServer.Addr + "/stream/" + rs.GetStreamID()
+	setLocalhost := func(addr string, prefix string) string {
```

Comment:
The local function declaration is a habit from FP languages :)

I originally had a couple other prefixes, but realized they weren't needed, so we just have one now. Sort of prefer a function over straight if-else, because it makes the intent of the code more descriptive, but can inline it here.